src/main/java/uk/gov/justice/digital/hmpps/keyworker/controllers/ControllerAdvice.java

```diff
@@ -54,7 +54,7 @@ public ResponseEntity<ErrorResponse> handleException(final AccessDeniedException
     @ExceptionHandler(AllocationException.class)
     public ResponseEntity<ErrorResponse> handleException(final AllocationException e) {
```

Comment:
i think you can leave this as  a 400. its a specific exception (AllocationException)